samples/imagefile_name_category_tags/Military-All-Icons.csv

```diff
@@ -4318,8 +4318,8 @@ filePath,pointSize,styleItemName,styleItemCategory,styleItemTags,styleItemUnique
 {Symbols_Root}\Legacy\2525C\I-U-SRS---.emf,64,Signal Intercept (Sea Subsurface) : Radar : Radar : Surface Search (Sea Subsurface),Signals Intelligence - Subsurface : Main Icon,Signals Intelligence - Subsurface;Signal Intercept (Sea Subsurface) : Radar : Radar : Surface Search (Sea Subsurface);I*UPSRS-----***;MAIN;Point;Signal Intercept (Sea Subsurface) : Radar : Radar : Surface Search (Sea Subsurface);I-U-SRS---,I-U-SRS---,Point,
 {Symbols_Root}\Legacy\2525C\I-U-SRT---.emf,64,Signal Intercept (Sea Subsurface) : Radar : Jammer : Target Acquisition (Sea Subsurface),Signals Intelligence - Subsurface : Main Icon,Signals Intelligence - Subsurface;Signal Intercept (Sea Subsurface) : Radar : Jammer : Target Acquisition (Sea Subsurface);I*UPSRT-----***;MAIN;Point;Signal Intercept (Sea Subsurface) : Radar : Jammer : Target Acquisition (Sea Subsurface);I-U-SRT---,I-U-SRT---,Point,
 {Symbols_Root}\Legacy\2525C\I-U-SRU---.emf,64,Signal Intercept (Sea Subsurface) : Radar : Radar : Unknown (Sea Subsurface),Signals Intelligence - Subsurface : Main Icon,Signals Intelligence - Subsurface;Signal Intercept (Sea Subsurface) : Radar : Radar : Unknown (Sea Subsurface);I*UPSRU-----***;MAIN;Point;Signal Intercept (Sea Subsurface) : Radar : Radar : Unknown (Sea Subsurface);I-U-SRU---,I-U-SRU---,Point,
-{Symbols_Root}\Legacy\2525B_Change_2\G-G-GPAM--.emf,64,Airspace Control Points : Marshall Point,Control Measure : Point,Control Measure;Airspace Control Points : Marshall Point;G*GPGPAM--****X;MAIN;Point;Airspace Control Points : Marshall Point;G-G-GPAM--,G-G-GPAM--,Point,
-{Symbols_Root}\Legacy\2525B_Change_2\G-G-GPAC--.emf,64,Airspace Control Points : Corridor Tab,Control Measure : Point,Control Measure;Airspace Control Points : Corridor Tab;G*GPGPAC--****X;MAIN;Point;Airspace Control Points : Corridor Tab;G-G-GPAC--,G-G-GPAC--,Point,
+{Symbols_Root}\Legacy\2525B_Change_2\G-G-GPAM--.emf,64,Airspace Control Points : Marshall (2525B),Control Measure : Point,Airspace Control Points;Marshall;G-G-GPAM--;LEGACY_ICON;MAIN;Point;Airspace Control Points : Marshall (2525B);G-G-GPAM--,G-G-GPAM--,Point,
```

Comment:
@csmoore : It would be useful for me to understand your guidelines for when you believe adding "(2525B)" to a name is warranted.  I *think* it's when 2525Bc2 and 2525C have the same SIDC for a symbol but draw them differently?